tools/teaktool/source/tlf.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Zlib
+//
+// Copyright (C) 2023 Antonio Niño Díaz
+
+#ifndef TLF_H__
+#define TLF_H__
+
+#include <stdint.h>
+
+// TLF = Teak Loadable Format
```

Comment:
Can we have a specification file for this format made?

Reply:
Done.